lib/potentialbuild.rb

```diff
@@ -693,8 +634,6 @@ def post_results(compiler, pending)
       'unhandled_failure' => !@failure.nil?,
       'build_error_count' => build_errors,
       'build_warning_count' => build_warnings,
-      'package_error_count' => package_errors,
-      'package_warning_count' => package_warnings,
```

Comment:
Removed several variables from the metadata that is sent up to the results repo.